### ISSUE TITLE
Cannot install CRD because of preserveUnknownFields: true

### ISSUE THREAD

**Issue by bsctl** [clastix/capsule-helm-chart#4]
With latest commit, the `preserveUnknownFields: true` has been added in Tenant CRD manifest.

The installation fails with following error:

```
$ kubectl apply -f helm/capsule/crds/tenant-crd.yaml 
The CustomResourceDefinition "tenants.capsule.clastix.io" is invalid: spec.preserveUnknownFields: Invalid value: true: cannot set to true, set x-preserve-unknown-fields to true in spec.versions[*].schema instead
```

Or using Helm:

```
$ helm install capsule helm/capsule --values helm/values.yaml
Error: failed to install CRD crds/tenant-crd.yaml: CustomResourceDefinition.apiextensions.k8s.io "tenants.capsule.clastix.io" is invalid: spec.preserveUnknownFields: Invalid value: true: cannot set to true, set x-preserve-unknown-fields to true in spec.versions[*].schema instead
```

Please, see here: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#field-pruning

**Comment by MaxFedotov** [comment]
closed by #11

**Comment by MaxFedotov** [comment]
closed by #11
